expected/17.0/extra/returning.out

```diff
@@ -0,0 +1,519 @@
+--SET log_min_messages  TO DEBUG4;
+--SET client_min_messages  TO DEBUG4;
+--Testcase 01:
+CREATE EXTENSION sqlite_fdw;
+--Testcase 02:
+CREATE SERVER sqlite_svr FOREIGN DATA WRAPPER sqlite_fdw
+OPTIONS (database '/tmp/sqlite_fdw_test/common.db');
+--Testcase 03:
+CREATE SERVER sqlite2 FOREIGN DATA WRAPPER sqlite_fdw;
+--Testcase 04:
+IMPORT FOREIGN SCHEMA main FROM SERVER sqlite_svr INTO public;
+--Testcase 05:
+SELECT * FROM "type_STRING";
+  col   
+--------
+ string
+ TYPE
+ type
+(3 rows)
+
+--Testcase 06:
+ EXPLAIN (VERBOSE, COSTS OFF)
+UPDATE "type_STRING" SET col = '_' || substr(col, 2) RETURNING *;
```

Comment:
You described "RETURNING have been introduced in SQLite 3.35.0", so we expect RETURNING clause will be pushed down to SQLite. Could you implement this point?

Reply:
I have begun in https://github.com/pgspider/sqlite_fdw/pull/107/commits/d339f8ce9fea91fe647ead4d79196472723968b1 , but removing a pushing down stopper is not enough, please see https://github.com/pgspider/sqlite_fdw/pull/107#issuecomment-2561576185.